src/Graph.ts

```diff
@@ -569,9 +569,10 @@ export default class Graph {
 				? Promise.resolve(resolvedId)
 				: this.resolveId(source, module.id)
 			).then(resolvedId => {
-				const externalId =
-					resolvedId ||
-					(isRelative(source) ? resolve(module.id, '..', source) : source);
+				// todo: there seems to be some issue here!
```

Comment:
@lukastaegert I might found some issue here, though I won't like to address it in the PR.